rest/v10/index.ts

```diff
@@ -1020,12 +1020,12 @@ export const CDNRoutes = {
 
 	/**
 	 * Route for:
-	 * - GET `/avatar-decorations/{user.id}/{user.avatar_decoration}.png`
+	 * - GET `/avatar-decoration-presets/{avatar_decoration_data_asset}.png`
 	 *
 	 * This route supports the extension: PNG
 	 */
-	userAvatarDecoration(userId: Snowflake, userAvatarDecoration: string) {
```

Comment:
The old function should stay, and be marked as deprecated in favour of the new one.
This change is also missing in rest/v9